spec/integration/issue8test.js

```diff
@@ -53,7 +53,37 @@ casper.test.begin('Testing About the Courses page', 13, function(test){
             return __utils__.findAll('.row h3 a').length >= 6;
         }, 'there are at least 6 course title links on the page');
     });
+    casper.then(function(){
```

Comment:
Please ignore #34 as I thought it would have conflicted with the current branch I'm working on. Ignore the other pull request.